content/en/blog/news/five-new-things/index.md

```diff
@@ -0,0 +1,114 @@
+
+```yaml
+apiVersion: core.spinkube.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: salutations
+spec:
+  image: "ghcr.io/spinkube/spin-operator/salutations:20241105-223428-g4da3171"
+  executor: containerd-shim-spin
+  replicas: 1
+  components:
+    - hello
+```
+
+We're really excited about this feature because it makes developing microservices easier. Locally keep your whole app together and develop it as a monolith. When you go to production selectively split your app based on the characteristics of each component. For example you might run one component on nodes with GPUs and another on nodes that are closer to the end user.
+
+If you want to learn more about how to use selective deployments in SpinKube checkout this [tutorial](TODO).
+
+## OpenTelemetry Support
+
+Spin has [supported OpenTelemetry](https://www.fermyon.com/blog/spin-v24#experimental-support-for-opentelemetry-observability) for a while now, and it's now available in SpinKube. OpenTelemetry is an observability standard that makes understanding your applications running production much easier via traces and metrics.
+
+To configure a `SpinApp` to send telemetry data to an OpenTelemetry collector you need to modify the `SpinAppExecutor`.
+
+```yaml
+apiVersion: core.spinkube.dev/v1alpha1
+kind: SpinAppExecutor
+metadata:
+  name: otel-shim-executor
+spec:
+  createDeployment: true
+  deploymentConfig:
+    runtimeClassName: wasmtime-spin-v2
+    installDefaultCACerts: true
+    otel:
+      exporter_otlp_endpoint: http://otel-collector.default.svc.cluster.local:4318
+```
+
+Now any Spin apps using this executor will send telemetry to the collector at `otel-collector.default.svc.cluster.local:4318`. For full details on how to use OpenTelemetry in SpinKube checkout this [tutorial](/docs/topics/monitoring-your-app).
+
+![OpenTelemetry](/otel.png)
+
+## MQTT Trigger Support
+
+The ContainerD Shim for Spin has added support for [MQTT triggers](https://github.com/spinkube/spin-trigger-mqtt). [MQTT](https://mqtt.org/) is a lightweight, publish-subscribe messaging protocol that enables devices to send and receive messages through a broker. It's used all over the place to enable Internet of Things (IoT) designs.
+
+If you want to learn more about how to use this new trigger checkout this [blog post](https://www.fermyon.com/blog/mqtt_trigger_spinkube) by Kate Goldenring.
+
+## Spintainer Executor
+
+In SpinKube there is a concept of an executor. An executor is defined by the `SpinAppExecutor` CRD and it configures how a `SpinApp` is executed. Typically you'll want to define an executor that uses the ContainerD shim for Spin.
+
+```yaml
+apiVersion: core.spinkube.dev/v1alpha1
+kind: SpinAppExecutor
+metadata:
+  name: containerd-shim-spin
+spec:
+  createDeployment: true
+  deploymentConfig:
+    runtimeClassName: wasmtime-spin-v2
+    installDefaultCACerts: true
+```
+
+However, it can also be useful to run your Spin application directly in a container. You might want to:
```

Comment:
Tried to tie the executor concept in a bit more before we talk about the benefits of containerization. Let me know if that resonates with you. 

```suggestion
This configurable executor affords flexibility of running your Spin application in different environments without having to make any modifications to the `SpinApp`. For example, you may want to achieve the following by running your Spin app in a container so that you can: